### DISCUSSION TITLE
希望能够去除视频详情页面新的推荐广告

### DISCUSSION THREAD

**Discussion by thevenomsnake** [the1812/Bilibili-Evolved#3288] [功能建议 / Ideas]
div data-loc-id="4330" class="video-ad-creative-card report-wrap-module report-scroll-module"><div class="card-box"><div class="pic-box"><div class="pic"

会附着在“接下来播放”上方

**Comment by Geeyun-JY3** [comment]
请
善
用
搜
索
#3269

**Reply by thevenomsnake** [reply]
嗯，我搜索的是 “推荐广告 ”，发现没有相关，没有想到他们用的关键字和我想得不一样，真是抱歉呢😊
